components/jsonpath.rst

```diff
@@ -0,0 +1,94 @@
+The JsonPath Component
+==================
+
+    The JsonPath component queries JSON strings using JSONPath syntax.
+
+What is It?
+-----------
+
+The Symfony JsonPath component parses JSON strings to retrieve specifics values.
+
+`JSON`_, *JavaScript Object Notation*, is a lightweight data-interchange format. 
+It is easy for humans to read and write. JSON is a widely used format in REST API and data manipulation.
+
+.. tip::
+
+    Learn more about `JSONPath`_ specifications.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/json-path
+
+.. include:: /components/require_autoload.rst.inc
+
+Why?
+----
+
+Fast
+~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Easy to read
+~~~~~~~~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Write powerful tests
+~~~~~~~~~~~~~~~~~~~~
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+Vestibulum id velit turpis. Sed vitae nisl placerat, rutrum ex nec, porta ante.
+Maecenas condimentum cursus risus at fringilla.
+
+Ready-to-use functions
```

Comment:
You may first introduce basic queries, then talk about built-in functions in the advanced usages part.